### PR TITLE
Bump k8s provider to 1.30

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR bumps KUBEVIRT_PROVIDER and KUBEVIRTCI_TAG to more updated values.

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -356,7 +356,7 @@ make docker-build-registry
 make cluster-up
 
 # bridge up a local cluster with kubernetes 1.25
```
Comment (oshoval) [inline README.md:358]: please change this comment / either to 1.30 or just making it generic

Comment (RamLavi) [reply README.md]: DONE

**Comment by oshoval** [comment]
Thanks

/lgtm

**Comment by oshoval** [comment]
btw seems hack/bump-kubevirtci.sh is broken, at least locally

**Pull request merged**